backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Controllers/DelegationsController.cs

```diff
@@ -41,7 +41,7 @@ public DelegationsController(IPolicyAdministrationPoint policyAdministrationPoin
         /// <response code="500">Internal Server Error</response>
         [HttpPost]
         [Authorize(Policy = AuthzConstants.ALTINNII_AUTHORIZATION)]
-        [Route("authorization/api/v1/[controller]/AddRules")]
```

Comment:
Should perhaps update the casing of the operation names also?

Reply:
Yes. Fixing